Modules/System/Azure Blob Services API/src/Helper/ABSFormatHelper.Codeunit.al

```diff
@@ -8,19 +8,23 @@ codeunit 9044 "ABS Format Helper"
     Access = Internal;
 
     [NonDebuggable]
-    procedure AppendToUri(var Uri: Text; ParameterIdentifier: Text; ParameterValue: Text)
+    procedure AppendToUri(var UriText: Text; ParameterIdentifier: Text; ParameterValue: Text)
     var
+        Uri: Codeunit Uri;
         ConcatChar: Text;
         AppendType1Lbl: Label '%1%2=%3', Comment = '%1 = Concatenation character, %2 = Parameter Identifer, %3 = Parameter Value', Locked = true;
         AppendType2Lbl: Label '%1%2', Comment = '%1 = Concatenation character, %2 = Parameter Value', Locked = true;
+        EscapedParameterValue: Text;
+
     begin
         ConcatChar := '?';
-        if Uri.Contains('?') then
+        if UriText.Contains('?') then
             ConcatChar := '&';
+        EscapedParameterValue := Uri.EscapeDataString(ParameterValue);
         if ParameterIdentifier <> '' then
-            Uri += StrSubstNo(AppendType1Lbl, ConcatChar, ParameterIdentifier, ParameterValue)
+            UriText += StrSubstNo(AppendType1Lbl, ConcatChar, ParameterIdentifier, EscapedParameterValue)
         else
-            Uri += StrSubstNo(AppendType2Lbl, ConcatChar, ParameterValue)
+            UriText += StrSubstNo(AppendType2Lbl, ConcatChar, ParameterValue, EscapedParameterValue)
```

Comment:
Something is fishy here. You probably meant to replace the ParameterValue with the EscapedParameterValue. I'll fix that during check-in.

Reply:
You're totally right. This looks like a copy paste issue.
It's time, that we get the pipelines working again, then I would have seen this earlier.

Reply:
Indeed. We'll get those working again soon.